Release fleetd 1.39.0 (#26185)
fleetdm/fleet: orbit/CHANGELOG.md

## Orbit 1.39.0 (Feb 07, 2025)

> * Pass `SystemDrive` Environemt variable to osqueryd when present in Orbit

Reviewer: ```suggestion
* Passed `SystemDrive` Environemt variable to osqueryd when present in Orbit
```

* Removed rollback to connect to Fleet's old TUF server (added in case 1.38.0 had an emergency bug).

* Added bash interpreter support for script execution

* Fixed a bug where fleetd was not properly cleaning up temporary directories during software
  installation.

> Fleetd can now download software installers from signed CDN URLs.

Reviewer: ```suggestion
* Allowed Fleetd to download software installers from signed CDN URLs.
```
